ui/components/app/collectible-default-image/collectible-default-image.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from '@material-ui/core';
+import classnames from 'classnames';
+import { TYPOGRAPHY } from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export default function CollectibleDefaultImage({
+  name,
+  tokenId,
+  handleImageClick,
+}) {
+  const t = useI18nContext();
+  return (
+    <div
+      className={classnames('collectible-default__wrapper', {
+        'collectible-default__clickable': handleImageClick,
+      })}
+      onClick={handleImageClick}
+    >
+      <img className="collectible-default__image" src="images/empty-nft.png" />
```

Comment:
Instead of adding an image could we use a background color instead? That should save us a few Bytes 🔍  Could we add `background-color: var(--color-background-alternative);` to `.collectible-default__wrapper`?

Reply:
I tried this at first but had trouble keeping the div retain the correct size/aspect ratio across different breakpoints without using some cheap padding tricks. But I'm not a CSS wizard so I'm sure you know better how to achieve this without using an img!

Reply:
Addressed https://github.com/MetaMask/metamask-extension/pull/15034/commits/a83bcd39ea3373d179051116dd9ee5009fde5788